### PR TITLE
#585 #562 #586 #589 #597 #598 #592 #600

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**
#585 #562 #586 #589 #597 #598 #592 #600

### CODE REVIEW

File: src/routes/CaseStudies/CaseStudies.js [hunk 1]
```diff
@@ -14,7 +15,7 @@ import has from './assets/has.jpg'
 import booksley from './assets/booksley.jpg'
 import sporty from './assets/sporty.jpg'
 
-export const CaseStudies = pure(() => (
+export const CaseStudies = pure(({ lessThanMedium }) => (
```
Comment (ArekRado) [inline src/routes/CaseStudies/CaseStudies.js]: do not use medium break point (use large and small)

File: src/routes/CaseStudies/CaseStudies.module.scss [hunk 2]
```diff
@@ -21,15 +21,18 @@
 
   @include media-breakpoint-down(md) {
     @include font(h4, $color-gray-dark, $font-family: $font-family-light);
-    line-height: 34px;
+    line-height: 34px !important;
     font-weight: 300;
   }
 
   .titleBold {
```
Comment (ArekRado) [inline src/routes/CaseStudies/CaseStudies.module.scss]: do you need nested classes?

File: src/routes/CaseStudies/index.js [hunk 3]
```diff
@@ -1 +1,6 @@
-export { CaseStudies } from './CaseStudies'
+import { connect } from 'react-redux'
+import { CaseStudies as CaseStudiesComponent } from './CaseStudies'
+
+const mapStateToProps = ({ browser }) => ({ lessThanMedium: browser.lessThan.medium })
```
Comment (ArekRado) [inline src/routes/CaseStudies/index.js]: move connect to CaseStudies file to avoid magic in index

Comment (szczepanbarszczowski) [reply src/routes/CaseStudies/index.js]: +1

**Pull request merged**